nft_ingester/src/bin/burnt_assets_ingester/main.rs

```diff
@@ -0,0 +1,453 @@
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+struct Args {
+    /// Path to the target RocksDB instance
+    #[arg(short, long, env = "TARGET_MAIN_DB_PATH")]
+    target_db_path: PathBuf,
+
+    /// Path to the NDJSON file with burnt assets
+    #[arg(short, long)]
+    input_file: PathBuf,
+
+    /// Batch size for storing assets (default: 1000)
+    #[arg(short = 'b', long, default_value_t = 1000)]
+    batch_size: usize,
+}
+
+fn convert_asset_to_complete_details(asset: Asset) -> AssetCompleteDetails {
+    let pubkey = Pubkey::try_from(asset.id.as_str()).expect("Invalid pubkey in asset id");
+
+    // Convert ownership data
+    let owner = AssetOwner {
+        pubkey,
+        owner: Updated {
+            value: Some(
+                Pubkey::try_from(asset.ownership.owner.as_str()).expect("Invalid owner pubkey"),
+            ),
+            update_version: None,
+            slot_updated: 0,
+        },
+        delegate: Updated {
+            value: asset
+                .ownership
+                .delegate
+                .as_ref()
+                .map(|d| Pubkey::try_from(d.as_str()).expect("Invalid delegate pubkey")),
+            update_version: None,
+            slot_updated: 0,
+        },
+        owner_type: Updated {
+            value: OwnerType::from(asset.ownership.ownership_model),
+            update_version: None,
+            slot_updated: 0,
+        },
+        owner_delegate_seq: Updated { value: None, update_version: None, slot_updated: 0 },
+        is_current_owner: Updated { value: true, update_version: None, slot_updated: 0 },
+    };
+
+    // Convert static details
+    let static_details = Some(AssetStaticDetails {
+        pubkey,
+        specification_asset_class: SpecificationAssetClass::from(asset.interface),
+        royalty_target_type: asset.royalty.as_ref().map_or(RoyaltyTargetType::Unknown, |r| {
+            RoyaltyTargetType::from(r.royalty_model.clone())
+        }),
+        created_at: 0,
+        edition_address: None,
+    });
+
+    // Convert dynamic details
+    let dynamic_details = Some(AssetDynamicDetails {
+        pubkey,
+        is_compressible: Updated {
+            value: asset.compression.as_ref().map(|c| c.eligible).unwrap_or(false),
+            update_version: None,
+            slot_updated: 0,
+        },
+        is_compressed: Updated {
+            value: asset.compression.as_ref().map(|c| c.compressed).unwrap_or(false),
+            update_version: None,
+            slot_updated: 0,
+        },
+        is_frozen: Updated { value: asset.ownership.frozen, update_version: None, slot_updated: 0 },
+        supply: asset.supply.as_ref().map(|s| Updated {
+            value: s.print_current_supply,
+            update_version: None,
+            slot_updated: 0,
+        }),
+        seq: None,
+        is_burnt: Updated { value: asset.burnt, update_version: None, slot_updated: 0 },
+        was_decompressed: None,
```

Comment:
why do we hardcode it? what if some asset was decompressed and then burned? I believe it's very low chance that this asset exist but just to make sure

Reply:
No decompressed data in the assets that we have